docs/slimmer_templates.md

```diff
@@ -10,12 +10,20 @@ This is the same as the `gem_layout` template, except that this layout **doesn't
 
 Use this layout if you want to have full width content - such as the blue welcome bar on the GOV.UK homepage.
 
-## `gem_layout_account`
+## `gem_layout_account` (deprecated – use `gem_layout_account_manager` instead)
 
 This is a variation on the `gem_layout` template intended to be used on GOV.UK Account pages. This layout **doesn't** include the feedback component as the account pages use a different one from the rest of GOV.UK.
 
 This also includes the Account product name in the layout header and changes the logo link to the account homepage link.
 
+## `gem_layout_account_manager`
+
+This is intended as a skeleton wrapper for account pages. It is intended to be used when we need a page rendered by another frontend app to look like a page from `govuk-account-manager-prototype`.
+This layout omits the default feedback component for GOVUK as the account pages use a different one from the rest of GOV.UK. Instead it introduces an account-specific phase banner, account feedback prompt, and an account nav component. It also purposefully omits the global bar and user satisfaction survey bar. 
```

Comment:
Nitpick: I think there should be a line space between this and the previous paragraph?

Reply:
thanks, have updated!